src/main/java/frc/team4909/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,37 @@
+package frc.team4909.robot.subsystems.elevator;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+import edu.wpi.first.wpilibj.PWMVictorSPX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class ElevatorSubsystem extends Subsystem{
+    PWMVictorSPX leftElevatorSPX, rightElevatorSPX;
+    PWMTalonSRX leftElevatorSRX, rightElevatorSRX, pivotSRX;
+    Encoder pivotEncoder, liftEncoder1, liftEncoder2;
+    DigitalInput topLimitSwitch, bottomLimitSwitch;
+
+    public void ElevatorSubsytem(){
+        //Intake Pivot
+        pivotSRX = new PWMTalonSRX(3);
+        pivotEncoder = new Encoder(1, 2);
+
+        //Lift
+        leftElevatorSPX = new PWMVictorSPX(4);
+        rightElevatorSPX = new PWMVictorSPX(5);
+        leftElevatorSRX = new PWMTalonSRX(6);
+        rightElevatorSRX = new PWMTalonSRX(7);
+        liftEncoder1 = new Encoder(3, 4);
```

Comment:
There's only one encoder now, but it'll be wired directly to the SRX, not the Rio.

Reply:
(the design was changed to a single quad-motor gearbox instead of two dual-input gearbox)